transmonee_dashboard/src/transmonee_dashboard/pages/child_protection.py

```diff
@@ -103,17 +92,17 @@
                     ),
                     "compare": "Sex",
                 },
-                "line": {
-                    "options": dict(
-                        x="TIME_PERIOD",
-                        y="OBS_VALUE",
-                        color="Geographic area",
-                        hover_name="Geographic area",
-                        line_shape="spline",
-                        render_mode="svg",
-                    ),
-                    "trace_options": dict(mode="lines+markers"),
-                },
+                # "line": {
+                #     "options": dict(
+                #         x="TIME_PERIOD",
+                #         y="OBS_VALUE",
+                #         color="Geographic area",
+                #         hover_name="Geographic area",
+                #         line_shape="spline",
+                #         render_mode="svg",
+                #     ),
+                #     "trace_options": dict(mode="lines+markers"),
+                # },
```

Comment:
In general it is better to just delete code rather then comment big blocks of code